src/main.cpp

```diff
@@ -67,6 +67,8 @@ int nScriptCheckThreads = 0;
 bool fExperimentalMode = false;
 bool fImporting = false;
 bool fReindex = false;
+std::atomic<size_t> nSizeReindexed(0);   // valid only during reindex
+std::atomic<size_t> nFullSizeToReindex(1);   // valid only during reindex
```

Comment:
As these are only consumed by the metrics screen, define these in `metrics.cpp` alongside the other metrics globals. That also makes it easier to refactor the metrics screen later on to reduce globals.